neo.UnitTests/Ledger/UT_MemoryPool.cs

```diff
@@ -33,6 +50,14 @@ public void TestSetup()
             _unit.VerifiedCount.ShouldBeEquivalentTo(0);
             _unit.UnVerifiedCount.ShouldBeEquivalentTo(0);
             _unit.Count.ShouldBeEquivalentTo(0);
+            _unit2 = new MemoryPool(TheNeoSystem, 0);
+            plugin = new TestIMemoryPoolTxObserverPlugin();
```

Comment:
We should have events in order to prevent this workarounds

Reply:
Not quite understand about this comment. What does the workarounds refer to?

Reply:
Because i thought that was used for listen the mempool, not only for coverage.